src/moduler/informasjon/informasjon-henting.js

```diff
@@ -13,13 +13,21 @@ import { selectErVeileder } from '../identitet/identitet-selector';
 import * as AppPT from '../../proptypes';
 import { INFORMASJON_MODAL_VERSJON } from './informasjon-modal';
 import { STATUS } from '../../ducks/utils';
+import { setBackPath } from './informasjon-reducer';
+
+const redirectPath = '/informasjon';
 
 class RedirectTilInformasjon extends Component {
     componentWillMount() {
-        const { erVeileder, history } = this.props;
-        if (!erVeileder) {
-            history.push('/informasjon');
+        const { erVeileder, history, location, setBack } = this.props;
+        const path = location.pathname;
+
+        if (path === redirectPath || erVeileder) {
```

Comment:
Det fungere dette, jeg testet å gå path `/dialog` Og logikken for å sjekke det er er her 👍 syns dette bra.